README.md

```diff
@@ -71,45 +71,31 @@ The available functions are:
   - `destination`: The path where to store the snapshot. The default will be a `.heapsnapshot` in the current directory.
   - `runGC`: If to run the garbage collector before taking the snapshot. The default is `false` and it is ignored if the process is not started with the `--expose-gc` flag.
 
-- `generateHeapSamplingProfile([options], [callback]): [Promise]`: Generates a heap sampling profiler. The valid options are:
+- `generateHeapSamplingProfile([options], [callback]): [Promise]`: Starts generating a heap sampling profiler. The valid options are:
 
   - `destination`: The path where to store the profile. The default will be a `.heapprofile` in the current directory.
   - `interval`: Sample interval, in bytes. Default is `32768` (32KB).
-  - `duration`: Sample duration, in milliseconds. Default is `10000` (10 seconds).
-  - `signal`: the [AbortController](http://npm.im/abort-controller) `signal`.
+  - `duration`: Sample duration, in milliseconds. Default is `10000` (10 seconds), and it is ignored if `signal` is provided.
+  - `signal`: The [AbortController](http://npm.im/abort-controller) `signal` to use to stop the operation.
+
+  The function accepts a callback function, otherwise it returns a Promise. The resolved value (or the callback argument) will be
```

Comment:
You have this on line 91 below. Did you forget to remove this one?

Reply:
Nope. I know it's a little confusing, but these are two different tools which accept the almost the same arguments.
Thanks for noticing tho.